keyserver/coniksserver/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"os"
+	"os/signal"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/keyserver"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+const ConfigFile = "config.toml"
+const PoliciesFile = "policies.toml"
```

Comment:
I and @masomel  have had a discussion over this (https://github.com/coniks-sys/coniks-go/commit/b5907c7e2faa2cff4938056e20489ac40a401798#commitcomment-18595647). She pointed out for me that policy file was redundant.  I probably should remove this `policies.toml`. Thank you!